src/config/env.js

```diff
@@ -20,3 +20,12 @@ module.exports = Object.freeze({
     admin_email: process.env.ADMIN_EMAIL,
     admin_password: process.env.ADMIN_PASSWORD,
 });
+
+module.exports = Object.freeze({
+    ...config,
+    db_uri: config.db_uri ||
+        (config.db_host
+            && config.db_port
+            && config.db_name
+            && `mongodb://${config.db_host}:${config.db_port}/${config.db_name}`),
```

Comment:
The uri that was being used in the mongoose loader did not use the db_name directly, but instead used it in the options for the connection. Please change this so that the db_name is not used here.

Uses that desire to know the db_name shall fetch it as they need to, but db_uri should just be the uri to the mongodb database.

Reply:
There is no case where you need the URI without the db_name, I updated the mongo connection call to pass the complete URI instead of a db_name param in the options which has the same effect. The other cases always use the complete URI. Which makes sense, in the way they are connecting to a specific db instance, not to the server where the db is located

Reply:
I believe that we found out that this is not the case for the test environment :sweat_smile: